src/heap/src/Common.c

```diff
@@ -231,7 +231,9 @@ DEFINE_HEAP_ALLOC(commonHeapAlloc)
     CHK_ERR(pHeap->heapLimit != 0, STATUS_HEAP_NOT_INITIALIZED, "Heap has not been initialized.");
 
     // Check if we can allocate the chunk taking into account the allocation header and footer
-    overallSize = pBaseHeap->getAllocationHeaderSizeFn() + size + pBaseHeap->getAllocationFooterSizeFn();
+    overallSize = pBaseHeap->getAllocationHeaderSizeFn(size) +
```

Comment:
are getAllocationHeaderSizeFn and getAllocationFooterSizeFn taking param?

Reply:
Yes indeed, those don't take size. Will change